settings/options.html

```diff
@@ -1,15 +1,14 @@
 <!DOCTYPE html>
-  <head>
-    <meta charset="utf-8">
-    <link rel="stylesheet" href="options.css"/>
-  </head>
-  <body>
-    <form>
-      <label>Mario mode<input type="checkbox" id="mario"></label>
-      <br><button type="submit">Save</button>
-    </form>
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="options.css"/>
+</head>
+<body>
+  <form>
+    <label>Mario mode<input type="checkbox" id="mario"></label>
```

Comment:
3. To get the fancy styling I mentioned above, we need to put the input element before the label (instead of inside it), and add a `for="mario"` to the label, so that it knows what checkbox it's associated with.